detect_secrets/core/secrets_collection.py

```diff
@@ -85,9 +86,24 @@ def _load_baseline_from_dict(cls, data):
         :raises: IOError
         """
         result = SecretsCollection()
-        if 'results' not in data or 'exclude_regex' not in data:
+        if not all(key in data for key in (
+            'exclude_regex',
+            'results',
+            'plugins_used'
```

Comment:
This is gonna be a "breaking" change, because `plugins_used` was only introduced "recently". Then again, as long as we bundle this with https://github.com/Yelp/detect-secrets/pull/38 before wrapping it up as a version, it should be OK.

Reply:
Added the requirement of `version` in my merge with master, which *definitely* breaks (3 months ago plugins_used was added vs. version just added today.) 👍